layouts/page.html.twig

```diff
@@ -1,59 +1,54 @@
 <!DOCTYPE html>
-<html lang="en" class="no-js">
-
+<html lang="fr">
 <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>{{page.title}}</title>
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{{page.title}} - CYPHT</title>
     <link rel="icon" type="image/png" href="/img/icon.ico">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
```

Comment:
I am pleased by the use of Bootstrap (also used by Cypht the software)

I am not a fan of counting on external ressources but given this is a public website, it could be acceptable.

Why this CDN and not others?

Reply:
OK, good. That site also indicates:

```
When using CDN links, be sure to use the integrity attribute to verify the correct files and versions. These hashes are unique to each file and version of Bootstrap, so when you update to a new version, be sure the integrity attribute is also updated.

We also include a crossorigin="anonymous" attribute to prevent [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) errors.
```

Are we doing this?